shared/bash_remediation_functions/replace_or_append.sh

```diff
@@ -71,7 +71,7 @@ function replace_or_append {
 
   # If the key exists, change it. Otherwise, add it to the config_file.
   if `grep -qi $key $config_file` ; then
-    $sed_command "s/$key.*/$formatted_output/g" $config_file
+    eval $sed_command "s/$key.*/$formatted_output/g" $config_file
```

Comment:
This is broken, I cannot reproduce behavior from comment above.
I don't know how I got to it, anyway, fix is proposed in #2207.